packages/apps/graph-client/src/components/chainweb/chain-block.tsx

```diff
@@ -0,0 +1,94 @@
+import { Block } from '../../__generated__/sdk';
+import { keyframes, styled } from '../../styles/stitches.config';
+import { Box } from '../box';
+import { Text } from '../text';
+
+import { TimeTicker } from './time-ticker';
+
+import { RocketIcon, TimerIcon } from '@radix-ui/react-icons';
+import React from 'react';
+
+const Container: any = styled('div', {
```

Comment:
can we abstract components/wrapper/styling to seperate files ex styles? i donno the current pattern but just a thought for reusability.

Reply:
I would be fine with doing this! I'm not sure if there is a current pattern we follow so I just left them in the same file. I think it could be cleaner to move them to their own file so I'll make this update 👍